prov/gni/src/gnix_vc.c

```diff
@@ -54,6 +54,17 @@
 #include "gnix_vector.h"
 #include "gnix_xpmem.h"
 
+#if HAVE_ATOMICS
```

Comment:
Shouldn't the configure step fail if we don't have atomics?  If so we should remove this #if, as well as the one in the bitmap code.